### PR TITLE
Return error when can't get conn to send ACK

### PR DESCRIPTION
**Pull request by billfort**
When the reader gets packets and needs to send ACK back to the writer, if can't find the right connection, throw ConnNotFoundError.

### CODE REVIEW

**Comment by yilunzhang** [comment]
Always run `yarn build` before commit

File: src/session.js [hunk 1]
```diff
@@ -219,6 +219,8 @@ export default class Session {
       let conn = this.connections.get(util.connKey(localClientID, remoteClientID));
       if (conn) {
         conn.sendAck(packet.getSequenceId());
+      } else {
+        throw new errors.ConnNotFoundError(util.connKey(localClientID, remoteClientID));
```
Comment (yilunzhang) [inline src/session.js]: The error message here contains only connKey. It's better to be more verbose, like `Conn xxx not found`

Comment (billfort) [reply src/session.js]: Yes, modified.

**Comment by yilunzhang** [comment]
You forgot `yarn build` again...

**Pull request merged**